guides/running-airflow-workflow.md

```diff
@@ -0,0 +1,53 @@
+# Running A Workflow On Airflow
+This guide will walk you through how to deploy a workflow on your existing Airflow cluster.
+
+## Prerequisites
+- The Aqueduct metadata storage must be changed to an AWS S3 bucket. See [here](changing-metadata-store.md) for how to do so. The AWS credentials file for this
+bucket must be copied over to your Airflow cluster.
+- Modify your Airflow cluster's config to use `Basic Authentication`. See the instructions [here](https://airflow.apache.org/docs/apache-airflow/stable/security/api.html#basic-authentication) for how to do so. For now, Aqueduct only supports basic authentication
+using a username and password. We are working on adding support for other authentication methods.
+- Check that Airflow cluster is accessible from the machine running Aqueduct.
```

Comment:
How might someone do this?

Reply:
Actually, I'll remove this one, because this will be tested anyways when they try to connect the integration.